dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -957,6 +957,10 @@ ERROR_NO_DERIVED_PLATFORM=CWWKF1516E: The platform couldn't be determined. The f
 ERROR_NO_DERIVED_PLATFORM.explanation= The platform was not set or couldn't be derived from existing versioned features. 
 ERROR_NO_DERIVED_PLATFORM.useraction=Add a platform for versionless features.
 
+ERROR_VERSIONLESS_INSTALL=CWWKF1517E: The installFeature command does not support versionless features.
+ERROR_VERSIONLESS_INSTALL.explanation= The installFeature command does not support versionless features.
```

Comment:
Wondering if we could do something more than restate the error message. Something like "The feature that you tried to install using the installFeature command does not have a version specified in the (whatever place the command looks for a version number)." If the user can inspect that place prior to running the command next time, they can avoid hitting the error again.

Reply:
I would prefer to keep calling them versionless - because this is now an official type of feature, documented in greater detail. - I used some of your suggestions